data/peptides_hemolytic/meta.yaml

```diff
@@ -0,0 +1,143 @@
+---
+name: peptides_hemolytic
+description: |-
+    Hemolysis is referred to the disruption of erythrocyte membranes that decrease the life span of red blood cells and causes the release of Hemoglobin.
+    It is critical to identify non-hemolytic antimicrobial peptides as a non-toxic and safe measure against bacterial infections.
+    However, distinguishing between hemolytic and non-hemolytic peptides is a challenge, since they primarily exert their activity at the charged surface of the bacterial plasma membrane.
+    In this work, the hemolysis classifier is trained using data from the Database of Antimicrobial Activity and Structure of Peptides (DBAASP v3).
+    Hemolytic activity is defined by extrapolating a measurement assuming dose response curves to the point at which 50\% of red blood cells are lysed.
+    Activities below 100 mu g/ml, are considered hemolytic.
+    The data contains sequences of only L- and canonical amino acids.
+    Each measurement is treated independently, so sequences can appear multiple times.
+    This experimental dataset contains noise, and in some observations (40%), an identical sequence appears in both negative and positive class.
+    As an example, sequence "RVKRVWPLVIRTVIAGYNLYRAIKKK" is found to be both hemolytic and non-hemolytic in two different lab experiments (i.e. two different training examples). 
+targets:
+    - id: hemolytic
+      description: The ability of a peptide sequence to lyse red blood cells.
+      units:
+      type: boolean
+      names:
+          - noun: hemolytic activity
+          - noun: hemolysis
+          - verb: kill red blood cells
+          - verb: lyse red blood cells
+          - adjective: hemolytic
+      uris:
+benchmarks:
+    - name: 
+      link: 
+      split_column: split
+identifiers:
+    - id: sequence
+      type: sequence
```

Comment:
You also need the type `Other` because otherwise the checks (should) fail.
They available types you can find here: https://github.com/OpenBioML/chemnlp/blob/text_sampling/src/chemnlp/data_val/model.py#L9
Note: So far we don't have a peptide type, but if we get more data with it we can add it.